internals/generators/organization/config.yml.hbs

```diff
@@ -410,6 +435,7 @@ client:
       fields:
         id: logout_id
       additional_fields: []
+      logout_by_session: true
```

Comment:
can you please add a new prompt for this?
Something like: "Does your captive portal support log out by session ID?" yes/no (default to no).